src/inference/src/dev/converter_utils.cpp

```diff
@@ -723,6 +723,8 @@ class IAsyncInferRequestWrapper : public ov::IAsyncInferRequest {
     }
     void set_tensor(const ov::Output<const ov::Node>& port, const ov::Tensor& tensor) override {
         m_request->SetBlob(get_legacy_name_from_port(port), ov::tensor_to_blob(tensor._impl, m_unwrap_tensor));
+        if (tensor._so)
+            m_request->setPointerToSo(tensor._so);
```

Comment:
Why do you need to set infer request pointer from tensor?! Looks like you inverted the pipeline.

Reply:
here can find the failed test without this change:
https://openvino-ci.toolbox.iotg.sclab.intel.com/blue/organizations/jenkins/private-ci%2Fie%2Fie-tests-linux-ubuntu20-gpu/detail/ie-tests-linux-ubuntu20-gpu/79261/pipeline